packages/opc-footer/webpack.config.js

```diff
@@ -22,6 +22,10 @@ module.exports = {
             use: 'ts-loader',
             exclude: /node_modules/,
         },
+        {
+          test: /\.(png|jpe?g|gif|svg)$/i,
+          use: [{ loader: 'file-loader'}],
+        },
         {
             test: /\.css|\.s(c|a)ss$/,
             use: [{
```

Comment:
@gisumit Can we add CleanWebpackPlugin also?

Reply:
@hybridx why? what's the use case? is there any problem with the current webpack build config?

Reply:
It basically deletes the dist folder every time you run or build

Reply:
yeah added.